pkg/services/pluginsintegration/clientmiddleware/tracing_header_middleware.go

```diff
@@ -42,6 +46,22 @@ func (m *TracingHeaderMiddleware) applyHeaders(ctx context.Context, req backend.
 		}
 		req.SetHTTPHeader(headerName, gotVal)
 	}
+	m.applyTraceIDHeader(ctx, req)
```

Comment:
I'm pretty sure the tracing identifier is already sent in gRPC metadata via https://github.com/grafana/grafana/blob/4def287e62e7d51bfbc62e062d580aa575fc60cd/pkg/plugins/backendplugin/grpcplugin/client.go#L42-L51 so these changes shouldn't be needed.